tesseract_command_language/include/tesseract_command_language/move_instruction.h

```diff
@@ -98,10 +99,21 @@ class MoveInstruction
   std::string profile_{ DEFAULT_PROFILE_KEY };
 
   /** @brief The assigned waypoint (Cartesian or Joint) */
-  Waypoint waypoint_;
+  Waypoint waypoint_{ NullWaypoint() };
 
   /** @brief Contains information about the manipulator associated with this instruction*/
   ManipulatorInfo manipulator_info_;
+
+  friend class boost::serialization::access;
+  template <class Archive>
+  void serialize(Archive& ar, const unsigned int /*version*/)
+  {
+    ar& boost::serialization::make_nvp("move_type", move_type_);
```

Comment:
is `type_` captured in the base?

Reply:
Fixed. I removed the member variable type_ and put it in the `getType` function to avoid the need to serialize.